include/biosHandler.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+#include <sdbusplus/asio/connection.hpp>
+#include <sdbusplus/bus.hpp>
+
+#include <iostream>
+
+namespace vpd
+{
+/**
+ * @brief A class to operate upon and back up some of the BIOS attributes.
+ *
+ * The class initiates call backs to listen to PLDM service and changes in some
+ * of the BIOS attributes.
+ * On a factory reset, BIOS attributes are initialized by PLDM, this class waits
+ * until PLDM has grabbed a bus name before attempting any syncs.
```

Comment:
"until PLDM service grabs a bus name" ?

Reply:
Why you want this change? Both has the same meaning.

Reply:
I felt it need some tweak, or can we use 'acquired' instead of 'grabbed'.
if you feel the existing one is correct, its ok for me.